lib/ansible/modules/monitoring/zabbix/zabbix_action.py

```diff
@@ -40,7 +40,8 @@
     event_source:
         description:
             - Type of events that the action will handle.
-        required: true
+            - Required when C(state=present)
```

Comment:
Could you please add a changelog entry [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments) and  porting guide note [here](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/porting_guides/porting_guide_2.10.rst#id51)?